extension/mooltipass-content.js

```diff
@@ -1453,6 +1453,7 @@ cip.doSubmit = function doSubmit(pass)
 }
 
 cip.retrieveCredentialsCallback = function (credentials, dontAutoFillIn) {
+	if (!credentials) credentials = [];
```

Comment:
I thought so, didn't dare to skip what comes after just out of a hunch, but that's the way I would do it.